src/toil/cwl/cwltoil.py

```diff
@@ -213,31 +214,32 @@ def run(self, fileStore):
                                                   os.getcwd(), None,
                                                   outdir=outdir,
                                                   tmpdir=tmpdir,
-                                                  use_container=True)
+                                                  use_container=self.use_container)
```

Comment:
And maybe rename `kwargs` and `self.kwargs` to `(self.)executor_options or (self.)run_options`